packages/office-addin-manifest/src/manifestHandler/manifestHandler.ts

```diff
@@ -7,21 +7,20 @@ import * as fs from "fs";
 
 export abstract class ManifestHandler {
   /* eslint-disable no-unused-vars */
-  constructor() {
-    this.manifestPath = "";
+  constructor(manifestPath: string) {
+    this.manifestPath = manifestPath;
     this.fileData = "";
   }
 
-  public async readFromManifestFile(manifestPath: string): Promise<string> {
-    this.manifestPath = manifestPath;
+  public async updateFileData(): Promise<string> {
```

Comment:
updateFileData?  Looks like this is a read not a write.

Reply:
Yeah, it is a read, not a writer. The name got confusing
My idea is that this function updates the internal `fileData` variable, which is needed after making modifications to the manifest.
I will see if I can improve this design to avoid these confusions.